.github/pull_request_template.md

```diff
@@ -1,5 +1,9 @@
 <!--
 Hi there! Thanks for submitting a pull request to Concourse!
+
+Please ensure your PR title is appropriate because it will be used to generate
+the release note.
```

Comment:
May help to suggest what makes it 'appropriate' - that it should summarize the content of the PR?

Also, what are we going to do about naming conventions like `atc: structure:` or `feat(load_var):`?

Reply:
Ah good point, I can add in another sentence for describing what it means to be appropriate.

I was hoping to get rid of the naming conventions with the `atc:` or `feat(..)`! Maybe I should mention to not include any prefix within the title in the pr template?

Reply:
@clarafu Yeah, we could mention that. It's a bit redundant given the template already has a part where the user chooses bug fix / feature / etc., plus we already add labels.

Reply:
How does this sound 

```
Please ensure your PR title is appropriate because it will be used to generate
the release note. The title should try to briefly summarize the contents of the
PR and should be written as a sentence. Please refrain from adding any prefixes
within the title. An example of a PR title is "Added a new fly command
to disable resource versions".
```

is that a good enough example or should it not be in past tense?